packages/monaco/src/browser/monaco-diff-editor.ts

```diff
@@ -123,6 +128,22 @@ export class MonacoDiffEditor extends MonacoEditor {
     override shouldDisplayDirtyDiff(): boolean {
         return false;
     }
+
+    override handleVisibilityChanged(nowVisible: boolean): void {
+        if (nowVisible) {
```

Comment:
What's the reasoning for not calling `super()` somewhere?

Reply:
The `super` implementation operates on `this.editor` and `this.model`, and we could likely make that line up with one side or the other of the diff editor, but the only thing we actually have in common is extracting the model and assigning it to a field on the class because the types of the `setModel` call and the save/restore view state calls are different, so the `super` method doesn't do much work that we need to do, and its restoration / storage of one side likely isn't something that we want to do, since it didn't work to preserve the diff markers on the side that it was operating on before.